packages/config/maintenance/importConfig.ts

```diff
@@ -1532,9 +1556,21 @@ async function parseZWAProduct(
 	const manufacturerDir = path.join(processedDir, manufacturerIdHex);
 	await fs.ensureDir(manufacturerDir);
 
+	// Insert a TODO comment if necessary
+	let output = "";
+	if (
+		newConfig.devices.filter(
+			(d) => d.productType === "0x9999" || d.productId === "0x9999",
+		).length > 0 ||
+		newConfig.manufacturerIdHex === "0x9999"
+	) {
+		output = `// TODO: This file contains a placeholder for a productType, productID, or manufacturerId (0x9999) that must be corrected. 
+${JSONC.stringify(normalizeConfig(newConfig), null, "\t") + "\n"}`;
+	} else {
+		output = JSONC.stringify(normalizeConfig(newConfig), null, "\t") + "\n";
+	}
+
```

Comment:
I'd do
```
let output =...
```
And then conditionally prepend the comment:
```
if (...) {
output = "...\n" + output;
}
```

(check formatting yourself, I'm on mobile 🙃)